react/features/base/connection/actions.native.js

```diff
@@ -298,11 +298,12 @@ function _constructOptions(state) {
             const {
                 protocol,
                 hostname,
+                port,
                 contextRoot
             } = parseURIString(locationURL.href);
 
             // eslint-disable-next-line max-len
-            bosh = `${protocol}//${hostname}${contextRoot || '/'}${bosh.substr(1)}`;
+            bosh = `${protocol}//${hostname}:${port || '80'}${contextRoot || '/'}${bosh.substr(1)}`;
```

Comment:
This is incorrect. The port must consider the protocol too, so 80 for HTTP and 443 for HTTPS.

Reply:
yep, it may be like:

bosh = `${protocol}//${hostname}:${port ? ':' + port : ''}${contextRoot || '/'}${bosh.substr(1)}`;

Reply:
Yes, that looks better.